lib/puppet/util/command_line.rb

```diff
@@ -122,6 +123,12 @@ def run
                 $LOAD_PATH << dir unless $LOAD_PATH.include?(dir)
               end
 
+              if !Puppet[:disable_i18n]
```

Comment:
Seems like this should be used more frequently...

Reply:
Oh, yes, you're right. I'll add the others.